Sources/Datadog/DatadogConfiguration.swift

```diff
@@ -210,21 +210,51 @@ extension Datadog {
 
             /// Sets the hosts to be automatically traced.
             ///
-            /// Every request made to a traced host and its subdomains will create its Span with related information; _such as url, method, status code, error (if any)_.
+            /// This option **must be used together with** `DDURLSessionDelegate` **set as your** `URLSession` **delegate object**.
+            ///
+            /// Every request made to a traced host and its subdomains will create its `Span` with related information; _such as url, HTTP method, HTTP status code, error (if any)_.
             /// Example, if `tracedHosts` is `["example.com"]`, then every network request such as the ones below will be automatically traced and generate a span:
             /// * https://example.com/any/path
             /// * https://api.example.com/any/path
             ///
-            /// If your backend is also being traced with Datadog agents, you can see the full trace (e.g.: client → server → database) in your dashboard with our distributed tracing feature.
+            /// If your backend is traced with Datadog agent, you will see the full trace (e.g.: client → server → database) in your dashboard with our Distributed Tracing feature.
             /// A few HTTP headers are injected to auto-traced network requests so that you can see your spans in your backend as well.
             ///
-            /// If `tracedHosts` is empty, automatic tracing is disabled.
+            /// Until `tracedHosts` is set, automatic tracing is disabled.
+            ///
+            /// **NOTE 1:** Setting `tracedHosts` will install swizzlings on some methods of the `URLSession`. Refer to `URLSessionSwizzler.swift`
+            /// for implementation details.
             ///
-            /// **NOTE:** Non-empty `tracedHost`s will lead to modifying implementation of some `URLSession` methods, in case your app relies on `URLSession` internals please refer to `URLSessionSwizzler.swift` file for details
+            /// **NOTE 2:** Setting `tracedHosts`, but not using `DDURLSessionDelegate` will lead to inconsistent tracing of network requests.
```

Comment:
Yes, it doesn't start the span. But this is an implementation detail, which I didn't want to express in the public API comment. I prefer to say "don't use it this way, because **something** will go wrong", without defining the "something", as it may change. A good example is upcoming RUM Resources instrumentation - if delegate is not set, the RUM Resource will be still started, but never completed / send. WDYT?

Reply:
i see 👌 i meant it could be more direct such as _"without delegate, this won't work"_, that sounds more like a red flag.
either way works for me 👍

Reply:
Good for me, we can say it more bold:
```suggestion
            /// **NOTE 2:** Setting `tracedHosts`, without using `DDURLSessionDelegate` will make the tracing instrumentation not work.
```

Reply:
how about an active sentence instead of passive?
> ...Yet tracing instrumentation will *NOT* work without using `DDURLSessionDelegate`

we can ask a native speaker